### DISCUSSION TITLE
Ho can I configure devices to report water consumption?

### DISCUSSION THREAD

**Discussion by StayPirate** [astrandb/miele#156] [Q&A] [answered]
I'm the owner on a TWF760WP (dryer) and a WWD660 (washing machine).

I never configured water consumption in HA before (it's quite a new feature). I updated to astrandb/miele `v0.1.0` and when I try to add a new device to the energy dashboard I get no compatible devices to add, as shown from the following screenshot.

![image](https://user-images.githubusercontent.com/37073343/202032067-80ed0bb6-5ec0-44f2-b596-d31618c0c63c.png)

Does anyone successfully added any Miele device to the energy dashboard?

**Comment by astrandb** [comment, marked as answer]
You must enable the consumption sensors in the device screen for your washing machine. They are hidden in the Diagnostic card. If your machine supports consumption data it will start to report to the sensors when you are running a program. After two hours the statistics will be there. 
The sensors are disabled by default since they are not always functional in the API. There is unfortunately no way to know on beforehand if they work or not. The maintainer's experience is that sensors that report `Unavailable` or `unknown` raises more questions than hidden sensors do.

**Comment by M-Stege** [comment]
Hello,
I own a WMV960 and in the Miele app I can see the cost and consumption of water & energy in Home Assistant but the entities are not available.
![Screenshot](https://user-images.githubusercontent.com/84290446/204617374-ba08c8fd-0d50-4f22-9d13-0fbdb5584a38.jpg)
![Screenshot_1](https://user-images.githubusercontent.com/84290446/204618382-54546a7c-baeb-43f1-bc7d-196b8425bb46.jpg)

**Reply by astrandb** [reply]
Click on Energieverbrauch, open Advanced settings and select Activate.
Repeat for Wasserverbrauch.
Please see answer above.

**Reply by M-Stege** [reply]
Great
